pkg/autoconf/autoconf.go

```
@@ -0,0 +1,260 @@
+		selectTarget:     opts.SelectTarget,
+	}
+}
+
+func (a *autoconfRunner) Run(out io.Writer) error {
+	result, err := a.detector.Detect()
+	if err != nil {
+		return err
+	}
+
+	for _, c := range result.Configured {
+		fmt.Fprintf(out, "%s Secret %q already configured (%s).\n", greenCheck, c.ServiceName, formatLocations(c.Locations))
```

golangci-lint (2.11.4)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)